powerauth-push-server/src/test/resources/sql/batch-init-h2.sql

```diff
@@ -36,16 +36,13 @@ CREATE TABLE BATCH_JOB_EXECUTION  (
 ) ;
 
 CREATE TABLE BATCH_JOB_EXECUTION_PARAMS  (
-	JOB_EXECUTION_ID BIGINT NOT NULL ,
-	TYPE_CD VARCHAR(6) NOT NULL ,
-	KEY_NAME VARCHAR(100) NOT NULL ,
-	STRING_VAL VARCHAR(250) ,
-	DATE_VAL TIMESTAMP DEFAULT NULL ,
-	LONG_VAL BIGINT ,
-	DOUBLE_VAL DOUBLE PRECISION ,
-	IDENTIFYING CHAR(1) NOT NULL ,
-	constraint JOB_EXEC_PARAMS_FK foreign key (JOB_EXECUTION_ID)
-	references BATCH_JOB_EXECUTION(JOB_EXECUTION_ID)
+    JOB_EXECUTION_ID BIGINT NOT NULL ,
```

Comment:
🤔 `batch-init-h2.sql` is a testing script. How does it affect production? Shouldn't we include it in the migration guide and liquibase scripts?

Reply:
I will need to test this change for sure.

Reply:
I suggest we handle this DDL change separately, because it needs some analysis and we need to get the stack back to buildable state.